### ISSUE TITLE
Prognoz 

### ISSUE THREAD

**Issue by Aytuza76** [Aytuza76/Prognoz#1]
# predictor.py
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.ensemble import RandomForestClassifier
from sklearn.metrics import accuracy_score

def main():
    # Ma'lumotlarni yuklash
    data = pd.read_csv('tronpick_gemz_data.csv')

    # Ma'lumotlarni tayyorlash
    X = data[['feature1', 'feature2', 'feature3']]
    y = data['outcome']

    # Ma'lumotlarni o'qitish va test to'plamlariga ajratish
    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

    # Model yaratish
    model = RandomForestClassifier(n_estimators=100, random_state=42)
    model.fit(X_train, y_train)

    # Prognozlar qilish
    y_pred = model.predict(X_test)

    # Modelni baholash
    accuracy = accuracy_score(y_test, y_pred)
    print(f"Modelning aniqligi: {accuracy}")

    # Yangi ma'lumotlar uchun prognoz qilish
    sample_data = pd.DataFrame([[1.0, 2.0, 3.0]], columns=['feature1', 'feature2', 'feature3'])
    prediction = model.predict(sample_data)
    print(f"Yangi ma'lumot uchun prognoz: {prediction[0]}")

if __name__ == "__main__":
    main()
